doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -371,9 +385,38 @@ otherwise if `false`:
 * `<targetId>` the internal Neo4j id of target node
 * `<targetLabels>` a list of labels for target node
 
+==== Filter
+
+You can use Spark to filter properties of the relationship, the source node, or the target node. Just use the correct prefix:
+
+If `relationship.node.map` is set to **false**
+
+* ``\`source.[property]` `` for the source node properties
+* ``\`rel.[property]` `` for the relation property
+* ``\`target.[property]` `` for the target node property
+
+If `relationship.node.map` is set to **true**
+
+* ``\`<source>`.\`[property]` `` for the source node map properties
```

Comment:
why?

Reply:
If I got your question right: this is used to filter the source, rel, or target columns when are returned as a map (using the `relationship.node.map` == true). This is the syntax used to access properties inside said maps.

Reply:
but why needs to be a different syntax there, can't we figure that out internally? and use the appropriate adjusted filter. Just trying to understand.

Reply:
We have two ways to represent a relationship:
* with `relationship.node.map=false` => we have the nodes and relationship flattened so every node has the namespace `source/target` and the relationship has `rel`. This is the most descriptive representation;
* with `relationship.node.map=true` => we have the nodes represented as map, in order to compact the table representation.